orbital_core/utils/tracing.py

```diff
@@ -0,0 +1,149 @@
+import sys
```

Comment:
I feel like this whole file could be added to OpenCensus... thoughts on working on a PR for them?

or we could potentially task it in the future. It just seems like this type of integration is pretty standard for aiohttp apps, so it might make sense in an upstream project such as opencensus.

Reply:
Yes, this can be added to OpenCensus in the future.